.editorconfig

```diff
@@ -0,0 +1,9 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# 2 space indentation
+[*.rb]
+indent_style = space
+indent_size = 2
```

Comment:
What about adding also `trim_trailing_whitespace = true`?

- I'm not sure if we want to add it as a part of a maintenance update, maybe adding only to `master` would be better.
- It would make sense to have it in *all* repositories, but then we should agree on the exact content first (discuss at yast-devel@ ?)

Reply:
OK, I'll bring it up on the mailing list